app/views/templates/payments/paymentsHistoryCharge.scala.html

```diff
@@ -35,9 +38,17 @@
           </span>
         }
       </td>
-      <td class="numeric nowrap @if(transaction.amount < 0){repayment-money}" valign="top">@{positiveNegative(transaction.amount)}</td>
+      <td class="numeric nowrap
+        @if(transaction.amount < 0 &&
+              (!chargeType.name.equals(UnallocatedPayment.value) || !chargeType.name.equals(Refund.value))) {repayment-money}
```

Comment:
Logic should be if amount < 0 or is a Refund, make green bar appear (repayment-money)
Or if is Unallocated payment, make blue (unallocated-payment)